src/js/views/admin/list/clinicians-all_views.js

```diff
@@ -47,8 +47,15 @@ const ItemView = View.extend({
   },
   template: hbs`
     <td class="table-list__cell w-20">{{#unless name}}{{ @intl.admin.list.cliniciansAllViews.itemView.newClinician }}{{/unless}}{{ name }}</td>
-    <td class="table-list__cell w-40">{{#each groups}}{{#unless @first}}, {{/unless}}{{ this.name }}{{/each}}</td>
+    <td class="table-list__cell w-30">{{#each groups}}{{#unless @first}}, {{/unless}}{{ this.name }}{{/each}}</td>
     <td class="table-list__cell w-30"><span class="u-margin--r-8" data-access-region></span><span data-role-region></span></td>
+    <td class="table-list__cell w-20 {{#unless last_active_at}}table-list__cell--empty{{/unless}}">
+      {{#if last_active_at}}
```

Comment:
There is a way to set the value for null https://github.com/RoundingWell/RWell/blob/81f50ca2a61980ad000faaf20018d8658d92bba1/mainapp/assets/js/views/clinicians/clinicians/clinicians_views.js#L103
It's not _that_ much different, but I think I'd like to establish that pattern in case it is more useful in the future.

Reply:
Oh nice, that's much cleaner.